client/jest-setup.ts

```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom';
```

Comment:
A one line setup?

Reply:
Well, this just runs before every test file.
Also check out:  `@jest-environment jsdom` comment below in the test. That is more than just a comment, as it sets up jsdom for that file specifically :D

Reply:
Oh I see.
It is always confusing for me when comments are more than just comments.